Cartfile

```diff
@@ -1,5 +1,7 @@
 binary "https://building42.github.io/Specs/Carthage/iOS/Crashlytics.json"
 binary "https://building42.github.io/Specs/Carthage/iOS/Fabric.json"
+github "mapbox/mapbox-navigation-ios" ~> 0.33.0
```

Comment:
Consider using the latest version: `0.34.0`

Reply:
Fresh and hot 0.35.0 Navigation and 0.29.0 Directions are out!

Reply:
updated, but wasn't able to update version of Directions because the Navigation module depends on Directions 0.28.0